docs/index.md

```diff
@@ -3,14 +3,14 @@
   title: ""
 ---
 
-# AddressBook Level-3
+# Staff-Snap
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
This is the link for AB3; maybe can update it to our project? 😄

Reply:
Sounds good! 🐶